server/service_test.go

```diff
@@ -257,8 +257,8 @@ func TestGetHeader(t *testing.T) {
 		rr = backend.request(t, http.MethodGet, path, nil)
 		require.Equal(t, 2, backend.relays[0].GetRequestCount(path))
 		require.Equal(t, 2, backend.relays[1].GetRequestCount(path))
-		require.Equal(t, `{"code":502,"message":"no successful relay response"}`+"\n", rr.Body.String())
-		require.Equal(t, http.StatusBadGateway, rr.Code, rr.Body.String())
+		require.Equal(t, http.StatusNoContent, rr.Code)
+		// require.Equal(t, `{"code":502,"message":"no successful relay response"}`+"\n", rr.Body.String())
```

Comment:
Should we remove these commented codes? do we plan to uncomment them later? If we do, perhaps a TODO or some mentioning would be nice

Reply:
yes, i'll remove!